src/lib.rs

```diff
@@ -322,7 +339,7 @@ struct HoveredPanel {
 
 pub fn provide_drag_reorder<const COLUMNS: usize, E>(
     panel_order: [RwSignal<Vec<Oco<'static, str>>>; COLUMNS],
-) -> [NodeRef<E>; COLUMNS]
+) -> ([NodeRef<E>; COLUMNS], DragReorderContext)
```

Comment:
Returning the `DragReorderContext` is a nice refactor, giving the user more control over when they actually want to provide it as a context.

The name `provide_drag_reorder` becomes a little misleading though, implying it provides the context. Maybe we can refactor this to be `DragReorderContext::new`? Similar to leptos' `RwSignal::new`, etc.